lightning/src/util/events.rs

```diff
@@ -478,13 +483,20 @@ impl Writeable for Event {
 					(0, VecWriteWrapper(outputs), required),
 				});
 			},
-			&Event::PaymentForwarded { fee_earned_msat, claim_from_onchain_tx } => {
+			&Event::PaymentForwarded {
+				fee_earned_msat,
+				claim_from_onchain_tx,
+				from_channel_id,
+				to_channel_id,
+			} => {
 				7u8.write(writer)?;
 				write_tlv_fields!(writer, {
 					(0, fee_earned_msat, option),
 					(2, claim_from_onchain_tx, required),
+					(4, from_channel_id, option),
+					(6, to_channel_id, required)
```

Comment:
similar to comment above about tlv fields for HTLCUpdate -- plz help

Reply:
These would be 1 and 3. IIRC, later you may have an odd field become `required` once we decide to no longer support an older serialization format. This would allow us to drop `Option` from the interface. For example, `PaymentPathFailed` has an odd `Option<PaymentId>` because the event preexisted `PaymentId`, but at some point we could make the field simply `PaymentId` and stop supporting old serialization formats.